packages/project-access/src/file/file-search.ts

```diff
@@ -0,0 +1,91 @@
+import type { Editor } from 'mem-fs-editor';
+import { basename, dirname, join, sep } from 'path';
+import { default as find } from 'findit2';
+import { fileExists } from './file-access';
+
+/**
+ * Add missing dump properties
+ */
+declare module 'mem-fs-editor' {
```

Comment:
Add a brief comment about the need to do this?

Reply:
`mem-fs-editor` type do not expose `dump`, added it as comment.